### ISSUE TITLE
popup not working correctly after scrolling grid  

### ISSUE THREAD

**Issue by thankikaushik** [swimlane/ngx-datatable#1849]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**

The popup stops binding once we click any item after scrolling. Also noticed that binding takes place if i resize the window later on. Here for demo I have only bind the pre tag for easy understanding. 
**Expected behavior**

The modal popup should work same for all items. 
**Reproduction of the problem**

https://stackblitz.com/edit/gx-datatable
https://gx-datatable.stackblitz.io/
**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

windows 10, Angular 6 
* **Table version:** 

     "@swimlane/ngx-datatable": "11.3.2",
* **Angular version:** 

    "@angular/common": "6.0.0",
    "@angular/compiler": "6.0.0",
    "@angular/core": "6.0.0",
    "@angular/forms": "6.0.0",
    "@angular/platform-browser": "6.0.0",
    "@angular/platform-browser-dynamic": "6.0.0",
    "@angular/router": "6.0.0",
* **Browser:** [all ]

* **Language:** [all ]

**Comment by konuch** [comment]
Hello, as a workaround, maybe try calling `window.dispatchEvent(new Event('resize'));` right after modal has been displayed.

**Comment by thankikaushik** [comment]
@konuch  thank for solution, I can not use this as resize will bind the data but other third-party component like drop down will stop working on popup. so resize is not the solution,

**Comment by thankikaushik** [comment]
Finally the issue resolved by upgrading package version to 16.0.3.
